sui/transfer-object.js

```diff
@@ -0,0 +1,93 @@
+const { TransactionBlock } = require('@mysten/sui.js/transactions');
+const { Command, Option } = require('commander');
+const { printInfo, validateParameters } = require('../evm/utils');
+const { addExtendedOptions } = require('./cli-utils');
+const { getWallet, printWalletInfo } = require('./sign-utils');
+const { loadSuiConfig } = require('./utils');
+
+async function processCommand(chain, options) {
+    const [keypair, client] = getWallet(chain, options);
+    await printWalletInfo(keypair, client, chain, options);
+    const recipient = options.recipient;
+    validateParameters({
+        isKeccak256Hash: { recipient },
+    });
+
+    let objectId;
+
+    if (options.objectId) {
+        objectId = options.objectId;
+        validateParameters({
```

Comment:
Both branches need to check the objectId, so do it at the end and avoid duplication